src/openloco/interop/hooks.cpp

```diff
@@ -676,6 +677,13 @@ void openloco::interop::register_hooks()
             return 0;
         });
 
+    register_hook(
+        0x00407FEE,
+        [](registers& regs) FORCE_ALIGN_ARG_POINTER -> uint8_t {
+            openloco::ui::set_cursor_pos(regs.eax, regs.ebx);
+            return 0;
+        });
+
```

Comment:
This hook can probably be removed now that `updateCursorPosition` (0x004C6E65) has been implemented.